packages/account/src/Components/file-uploader-container/__tests__/file-uploader.spec.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Just for my understanding, what was the reason for disabling this? instead of using type **any** in this file can we define types for the props?